core/src/apps/cardano/shelley/utils.py

```diff
@@ -0,0 +1,13 @@
+# https://en.wikipedia.org/wiki/Variable-length_quantity
+def variable_length_encode(i):
```

Comment:
I find type annotations useful, e.g. `i: int`.

Reply:
I would add the type annotations in another PR later, when I'll be checking all type annotations.